lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -48,7 +48,10 @@ sub new {
             die "$@ \n" if $@;
             $self->{db} = "$params->{db}"->new( { config => $config } );
         };
-        die "$@ \n" if $@;
+        if ($@) {
+            warn "Internal error #001: Failed to initialize the [$params->{db}] database backend module: [$@] \n";
+            die "Internal error #001 \n";
+        }
```

Comment:
Why not a call to handle_exception() here so that all cases are done in the same way?

Reply:
I second this.

Also, we cannot have the '#NNN' format. If you paste one of those into an issue on Github, Github mangles it into a link to an issue or a pull request. It's really completely unworkable.

Reply:
Fixed in dcf35a6